examples/Domains/README.md

```diff
@@ -1,20 +1,14 @@
-
-
 ## 'Domains Target Functions' Examples
 
 This folder contains examples of how to use RAT to analyse what are referred
 to as 'Domains TF' samples, which are those where an incoherent sum is required
-in order to calculate the reflectivity. As is the case for normal models, 
-it is possible to use either 'Standard Layers','Custom Layers' or 'Custom XY' models.
+in order to calculate the reflectivity. As is the case for normal reflectivity i.e. non-polarised TF, 
+it is possible to use either 'Standard Layers', 'Custom Layers' or 'Custom XY' models.
 
 The basic principle for all three is that there is an additional fittable 
-parameter group called 'domain ratio' (as compared to normal models). 
+parameter group called 'domain ratio' (as compared to normal reflectivity). 
 Then each actual measured contrast, along with the other usual parameters, 
-has a 'Domain Ratio' field. 
+has a 'domain ratio' field. 
 
 The method for defining domains varies between Standard Layers and Custom models,
```

Comment:
Hmm, maybe write either: standard layers (lower case), or include the inverted commas - 'Standard Layers' - to be consistent with elsewhere.